src/app/delegation_backend/src/signer.go

```diff
@@ -0,0 +1,20 @@
+package delegation_backend
+
+// TODO think of getting rid of -L flags with relative paths
+
+// #cgo LDFLAGS: -L../result
+// #cgo LDFLAGS: -lmina_signer -lm
+// #cgo CFLAGS: -w -I ../result/headers
+// #include "crypto.h"
```

Comment:
Should we get rid of these `cgo` and `include` comments?

Reply:
Nope, these are needed for `cgo`, a Go's compiler unit which is responsible for bridging Go code with C